### PR TITLE
[Rule Tuning] Remote System Discovery Commands

### PR DESCRIPTION
**Pull request by imays11**
## Issues

[T1018 Remote System Discovery #39](https://github.com/elastic/mitre-eval-2022/issues/39)
## Summary
- Added coverage for additional remote system discovery tools : 
  - nltest
  - dsquery
  - dsget
  - net

image

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by shashank-elastic — APPROVED** [review]
LGTM

File: rules/windows/discovery_remote_system_discovery_commands_windows.toml [hunk 1]
```diff
@@ -4,7 +4,11 @@ integration = ["endpoint", "windows"]
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
+<<<<<<< tune_remote_system_discovery_commands_windows
```
Comment (terrancedejesus) [inline rules/windows/discovery_remote_system_discovery_commands_windows.toml:7]: I think there are some git comments that stayed when you staged and commited.

Comment (imays11) [reply rules/windows/discovery_remote_system_discovery_commands_windows.toml]: should be fixed now

**Review by DefSecSentinel — APPROVED** [review]
LGTM

**Pull request merged**